<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductForm</class>
 <widget class="QWidget" name="AddProductForm">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>343</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Товар</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <item>
    <widget class="QLabel" name="lHeader">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Добавление товара</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leArticle">
     <property name="placeholderText">
      <string>Артикул</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leName">
     <property name="placeholderText">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbCategory">
     <property name="placeholderText">
      <string>Категория</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="lPurchasePrice">
       <property name="text">
        <string>Закупочная цена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dsbPurchasePrice">
       <property name="suffix">
        <string>₽ </string>
       </property>
       <property name="maximum">
        <double>2147483647.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <widget class="QLabel" name="lRetailPrice">
       <property name="text">
        <string>Розничная цена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dsbRetailPrice">
       <property name="suffix">
        <string>₽</string>
       </property>
       <property name="maximum">
        <double>2147483647.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
     <item>
      <widget class="QLabel" name="lStock">
       <property name="text">
        <string>Остаток</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbStock">
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="20,80">
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
